src/App.js

```diff
@@ -1,16 +1,54 @@
-import React from 'react';
+import React from "react";
+import Search from "./components/Search";
+import MovieResults from "./components/MovieResults";
+import Info from "./components/Info";
 
 class App extends React.Component {
-  constructor(){
-    super();
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      movies: [],
+      moreInfo: []
```

Comment:
It might make sense to initialise `moreInfo` to empty string if it will store a string